homeassistant/components/airvisual/manifest.json

```diff
@@ -4,6 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/airvisual",
   "requirements": ["pyairvisual==2022.12.1"],
+  "dependencies": ["airvisual_pro"],
```

Comment:
I'm assuming this is for the `from homeassistant.components.airvisual_pro import DOMAIN as AIRVISUAL_PRO_DOMAIN`

I'm not sure it saves you anything making it a dep and they will have to pay to have both integrations in memory if they only have non-pro devices.